resources/css/table.css

```diff
@@ -10,6 +10,21 @@ tbody tr:nth-child(odd) {
   background-color: rgba(80, 80, 80, .09);
 }
 
+tbody tr:not(.forum-post) {
+  outline: 1px solid transparent;
```

Comment:
`outline` shouldn't be an issue here, as `border-radius` isn't being used and it isn't supposed to break anything anyway.